docs/cloud/rahti/storage/persistent.md

```diff
@@ -0,0 +1,64 @@
+# Persistent volumes
+
+**Persistent volumes** are storage which persist during & after pod's lifetime.
+
+Persistent volumes in Rahti are stored in a resilient storage such as CEPH, NFS or
+GlusterFS. They are claimed by a pod using a **PersistentVolumeClaim**. When a
+new claim is made, this can mean that either an existing volume is claimed or a
+new one is created dynamically and given to the pod to use.
+
+There are two storage classes available in Rahti:
+
+ * *glusterfs-storage*. This kind of volume supports "Read Write Many" (RWX) storage, this means multiple nodes can mount it in read-write mode. This is the default class.
```

Comment:
We could add a description which is better to use when

Reply:
Yes, I am writing it now. In a nutshell I will say that gluster is flexible (can mount everywhere), and standard-rwo is faster but can only be mounted in one node.

Reply:
It is done